kibana.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "opendistroAnomalyDetectionKibana",
+  "version": "1.11.0.0",
+  "kibanaVersion": "7.9.1",
```

Comment:
Should we change to 7.10?

Reply:
This PR only did the job for migration, without the version upgrade. I think keep the Kibana version during migration is easier for testing.

Reply:
Yizhe has some changes for this, there will be a follow up PR with upgrade-related changes.

Reply:
Sure, make sense.